### ISSUE TITLE
Windows Defender detects freshly compiled Osiris as Win32/Contebrew.A!ml

### ISSUE THREAD

**Issue by Spamm00r** [danielkrupinski/Osiris#2786]
I pulled a git yesterday and compiled it. Shortly after injecting it into csgo, Windows Defender detected the dll as Win32/Contebrew.A!ml

What is happening here? Why is Osiris coming with malware?

**Comment by Cassiedee** [comment]
Osiris doesn't come with malware and is completely safe. Windows Defender just got a new definition update and is now marking a bunch of false positives. Just disregard it and add an exception to the compiled Osiris Dll.

**Comment by Teemoplays** [comment]
Hi
Win32/Contebrew.A!ml is adware me atleast when i compiled my osiris version like a month ago it didnot show any malware adware etc... 
I strongly recomend you to redownload it again (or not) and do everything you usually do, change some values and manke it undetected and after compile it again.
If it still shows that (even if not) look at the code for anything suspitious.
Usualy there are false positives but the only say that it is a win32 trojan not an adware and it should not say that if you are "making" the dll yourself on your computer

Thank you

**Comment by Spamm00r** [comment]
Yes I did make the dll myself from a freshly downloaded git just yesterday. I dont trust ready made dlls for these specific reasons. And therefore I'm surprised, that every dll I make ends up being detected as  Win32/Contebrew.A!ml .

And not only the dll itself, but also some tmp files that are created during runtime in the windows temp folder.

As the source code is open, it should be possible to find what is causing this detection as Win32/Contebrew.A!ml .

What worries me is that if WIndows defender is already detecting it as Win32/Contebrew.A!ml , what will VAC do? 

VAC will also be easily able to sig the dll.

**Comment by linuxer9** [comment]
It's a false positive report, any cheating software is detect by most antiviruses/antimalwares due to its injection nature and memory manipulation (as there is no legit software would behave like this).
there is nothing to worry about.
regarding VAC, it only bans you as a result of signature scan, so if you have a slightly modified binaries it won't instantly ban you, but VAC recognize detected methods (siggs suspicious running files) and flags your account and that increases your chances to get OW'd, then if you get convicted, it will add this file signature to a black list.

just use VAC bypass if you worry about vac detection.

**Comment by Spamm00r** [comment]
You don't have to inject the dll. Windows Defender will detect the file as a virus just by opening the folder that conatins the dll.

**Comment by mkot2** [comment]
defender is a joke
